.setup/options/others/remove-old-snap.sh

```diff
@@ -3,8 +3,7 @@
 # CLOSE ALL SNAPS BEFORE RUNNING THIS
 set -eu
 
-# shellcheck disable=SC2162
 snap list --all | awk '/disabled/{print $1, $3}' |
-    while read snapname revision; do
+    while read -r snapname revision; do
```

Comment:
By using read -r instead of read, you prevent any backslashes in the input from being interpreted as an escape character.